features/step_definitions/web_steps.rb

```diff
@@ -24,6 +24,8 @@
 require File.expand_path(File.join(File.dirname(__FILE__), "..", "support", "paths"))
 require File.expand_path(File.join(File.dirname(__FILE__), "..", "support", "selectors"))
 
+require 'pry'
```

Comment:
You would think that the multiple times this threw me into a panic during capstone would have been lesson enough. But, no.